lib/std/crypto/ghash.zig

```diff
@@ -18,12 +18,13 @@ pub const Ghash = struct {
     pub const mac_length = 16;
     pub const key_length = 16;
 
-    const pc_count = if (builtin.mode != .ReleaseSmall) 16 else 4;
-    const agg_2_treshold = 5;
+    const pc_count = if (builtin.mode != .ReleaseSmall) 16 else 2;
     const agg_4_treshold = 22;
     const agg_8_treshold = 84;
     const agg_16_treshold = 328;
 
+    const mul_algorithm = if (builtin.cpu.arch == .x86) .karatsuba else .textbook;
```

Comment:
Could you add a comment here matching your PR description?

Reply:
Done :)